src/test/java/com/revature/services/QuestionServiceTest.java

```diff
@@ -1,25 +1,24 @@
 package com.revature.services;
 
-import static org.junit.Assert.assertArrayEquals;
```

Comment:
I am almost positive these import removals will conflict with another team however we can deal with that when we get there.